packages/frontend/core/src/modules/workbench/view/workbench-link.tsx

```diff
@@ -18,6 +18,7 @@ export const WorkbenchLink = ({
   const handleClick = useCallback(
     (event: React.MouseEvent<HTMLAnchorElement>) => {
       event.preventDefault();
+      event.stopPropagation();
```

Comment:
Why stop propagation

Reply:
without it ctrl+click a page link will also trigger page open for the current view.

Reply:
Internal logic of blocksuite?

Reply:
yeah, when the link is clicked it will emit event through `docLinkClicked` and calls https://github.com/toeverything/AFFiNE/blob/308f3e4b722f15c7a77fffd22f8fb3ffb6617f2f/packages/frontend/core/src/pages/workspace/detail-page/detail-page.tsx#L159